### PR TITLE
PRMDR 598 SES feedback mailbox

### PR DESCRIPTION
**Pull request by NogaNHS**

### CODE REVIEW

**Review by joefong-nhs — APPROVED** [review]

File: infrastructure/modules/ses/main.tf [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+resource "aws_route53_record" "ndr_amazonses_verification_record" {
+  count   = var.enable ? 1 : 0
+  zone_id = var.zone_id
+  name    = aws_ses_domain_mail_from.ndr_mail_from[0].mail_from_domain
+  type    = "TXT"
+  ttl     = 600
+  records = ["v=spf1 include:amazonses.com -all"]
+}
+resource "aws_ses_domain_mail_from" "ndr_mail_from" {
```
Comment (abbas-khan10) [inline infrastructure/modules/ses/main.tf:29]: Space here

File: infrastructure/modules/ses/variable.tf [hunk 2]
```diff
@@ -0,0 +1,14 @@
+variable "domain" {
+  type = string
+}
+
+variable "zone_id" {
+  type = string
+}
+variable "from_mail_prefix" {
```
Comment (abbas-khan10) [inline infrastructure/modules/ses/variable.tf:8]: space here

File: infrastructure/modules/ses/main.tf [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  count  = var.enable ? 1 : 0
+}
+
+resource "aws_ses_domain_dkim" "ndr_dkin" {
+  domain = aws_ses_domain_identity.ndr_ses[0].domain
+  count  = var.enable ? 1 : 0
+
+}
+
+resource "aws_route53_record" "ndr_amazonses_dkim_record" {
```
Comment (abbas-khan10) [inline infrastructure/modules/ses/main.tf:12]: could we rename anything `amazonses` to just `ses` please?

Comment (NogaNHS) [reply infrastructure/modules/ses/main.tf:11]: Do you mean in the record itself? or just the name in terraform?
  records = ["${aws_ses_domain_dkim.ndr_dkin[0].dkim_tokens[count.index]}.dkim.amazonses.com"]
I think this needs to be amazonses

**Pull request merged**